src/components/FAIcon.js

```diff
@@ -3,14 +3,19 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 
-const FAIcon = ({ icon, component: Component='span', modifiers=[], ...props }) => {
-  const className = classNames(
+const FAIcon = ({ icon, component: Component='span', extraClassname='', modifiers=[], ...props }) => {
+  let className = classNames(
     'fa',
     'fas',
     'fa-icon',
     `fa-${icon}`,
     ...modifiers.map(mod => `fa-icon--${mod}`),
   );
+
+  if (extraClassname) {
+    className += ` ${extraClassname}`;
+  }
```

Comment:
no, use the functionality of the classnames lib.

```suggestion
  const className = classNames(
    'fa',
    'fas',
    'fa-icon',
    `fa-${icon}`,
    ...modifiers.map(mod => `fa-icon--${mod}`),
    extraClassname,
  );
```